### DISCUSSION TITLE
a branch for old nvidia cards

### DISCUSSION THREAD

**Discussion by kftar** [ptr727/PlexCleaner#236] [Ideas]
the ffmpeg failed on my system that nvenc API version required is 12, but found 11. And there are many cards with version as old as 9. Even with latest driver. Does new API bring any notable features?

Can you create a branch that use lower version of nvenc header for ffmpeg? The only latest compatible version is pre-3.0 2.10.16, but it's unusable because we cannot choose hardware acceleration.

Thanks in advance.

**Comment by kftar** [comment]
or alternatively, save a copy of ffmpeg v5 in the same image

**Reply by ptr727** [reply]
You can copy in any version of ffmpeg you want, just turn off auto updates if on Windows.

**Comment by kftar** [comment]
related issue: https://github.com/HaveAGitGat/Tdarr/issues/797
